stdlib/3/curses/__init__.pyi

```diff
@@ -1,9 +1,10 @@
 import _curses
 from _curses import *  # noqa: F403
+from typing import Callable, Any, Sequence, Mapping
 
 LINES: int
 COLS: int
 
 def initscr() -> _curses._CursesWindow: ...
-def start_color(): ...
-def wrapper(func, *args, **kwds): ...
+def start_color() -> None: ...
+def wrapper(func: Callable, *args: Sequence[Any], **kwds: Mapping[Any, Any]) -> None: ...
```

Comment:
The type for `*` and `**` arguments is the type of individual elements, not the type of the `args/kwds` variable. You probably just want both of them to be Any. Also, can you make the func argument be `Callable[..., Any]` to be explicit?

Reply:
Ah! I kept running myself in circles on that how that should be typed. 
```def wrapper(func: Callable[..., Any], *arg: Any, **kwds: Any) -> None: ...```